assets/prototype/domain/eventEditor/containers/mutations/datetimes/updateTicketCache.js

```diff
@@ -0,0 +1,30 @@
+import { queries } from '../../queries';
+const { GET_TICKETS } = queries;
+
+const updateTicketCache = ({ proxy, datetimeIn, datetimeId, remove = false }) => {
```

Comment:
Maybe it should be in a different directory, under `tickets` maybe?
or in a `cache` folder maybe?

Reply:
Yes, it will be extracted when we create mutators for tickets.